rpm/authselect.spec.in

```diff
@@ -35,6 +35,16 @@ supported by authselect.
 
 %package libs
 Summary: Utility library used by the authselect tool
+# Required by scriptlets
+Requires: %{_bindir}/awk
```

Comment:
I don't think any of these are needed. At least in the past there was a number of packages that were guaranteed to be in a buildroot, did this change?